src/setup.rs

```diff
@@ -103,15 +104,26 @@ fn setup_rust_toolchain(kani_dir: &Path) -> Result<String> {
 }
 
 /// Install into the pyroot the python dependencies we need
-fn setup_python_deps(kani_dir: &Path) -> Result<()> {
+fn setup_python_deps(kani_dir: &Path, os: &os_info::Info) -> Result<()> {
     println!("[4/6] Installing Kani python dependencies...");
     let pyroot = kani_dir.join("pyroot");
 
     // TODO: this is a repetition of versions from kani/scripts/setup/$OS/install_deps.sh
+    let pkg_versions = &["cbmc-viewer==3.2", "colorama==0.4.3"];
+
+    if os.os_type() == os_info::Type::Ubuntu
+        && *os.version() == os_info::Version::Semantic(18, 4, 0)
```

Comment:
Is there a risk there will be an 18.4.1?

Reply:
I think there is not a risk of that because I think it's already running on "18.4.4" and not picking that up. I believe what's happening is it's parsing "18.04" as if it were a version number.

I agree this is seems fragile but this unit test seems to confirm this is the expected behavior: https://github.com/stanislav-tkach/os_info/blob/master/os_info/src/linux/lsb_release.rs#L317-L318